yarn.lock

```diff
@@ -7,6 +7,53 @@
   resolved "https://registry.yarnpkg.com/@0no-co/graphql.web/-/graphql.web-1.1.2.tgz#9af8deaf3f236c1c6ee99cc5349051475e5dcc83"
   integrity sha512-N2NGsU5FLBhT8NZ+3l2YrzZSHITjNXNuDhC4iDiikv0IujaJ0Xc6xIxQZ/Ek3Cb+rgPjnLHYyJm11tInuJn+cw==
 
+"@aa-sdk/core@*", "@aa-sdk/core@^4.0.0-alpha.8", "@aa-sdk/core@^4.17.0", "@aa-sdk/core@^4.66.1":
```

Comment:
is it possible to successfully build with the existing `yarn.lock` file on the `moldy/v5-base` branch to avoid these changes?